src/ScottPlot4/ScottPlot.Tests/PlotTypes/Scatter.cs

```diff
@@ -5,6 +5,7 @@
 using System.Text;
 using System.Threading.Tasks;
 using ScottPlot;
+using System.Drawing;
```

Comment:
I would recommend a test for plots where once NaNs are removed there are segments of length 1 or of length 0. Judging from the code I don't think these should cause a problem (there's a check in `DrawLines` to make sure the array has length > 1), but this is the sort of thing that could break were the implementation to change.

Reply:
I say because I have done something similar where I was splitting on null, and not handling segments that were too small caused some shenanigans that often regressed when we changed how we handled the splitted (splat? or splut?) data.